pages/AdListPage/[id]/ContactPage/index.js

```diff
@@ -0,0 +1,83 @@
+import useSWR from "swr";
+import { fetcher } from "../../../../helpers/api";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function ContactPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const { data: ad, error } = useSWR(`/api/ads/${id}`, fetcher);
+
+  if (error) return <h1>...sorry cannot contactdata</h1>;
```

Comment:
Seems like there is something missing here:
```suggestion
  if (error) return <h1>...sorry cannot load contact data</h1>;
```